docs/locale/fr/LC_MESSAGES/architecture.po

```diff
@@ -17,11 +17,65 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.6.0\n"
 
-#: ../../architecture.rst:5
 msgid "Architecture"
 msgstr "Architecture"
 
-#: ../../architecture.rst:7
-msgid "<TODO>"
-msgstr "<TODO>"
+msgid "Overview"
+msgstr "Aperçu"
 
+msgid "Parsec is divided between a client (responsible for exposing data to the user and providing an encryption layer) and a server (storing the encrypted data and notifying clients about other users activity such as data modification or new sharing)."
+msgstr "Parsec est divisé en un client (responsable d'exposer les données à l'utilisateur ainsi que de gérer la couche cryptographique) et le serveur (servant à stocker les données chiffrées ainsi qu'à notifier les clients des actions des autres utilisateurs telles une modification d'un fichier ou un nouveau partage)."
```

Comment:
i think i might be wrong) that:
`des autres utilisateurs telles une modification `
should be written
`des autres utilisateurs telle une modification `

Reply:
Nope, for that one that was the correct form, I checked to be sure :)